os/fs/vfs/fs_open.c

```diff
@@ -97,14 +93,22 @@ int inode_checkflags(FAR struct inode *inode, int oflags)
 }
 
 /****************************************************************************
- * Name: open
+ * Name: vopen
  *
  * Description:
- *   Standard 'open' interface
+ *   vopen() is identical to 'open' except that it accepts a va_list
+ *   as an argument versus taking a variable length list of arguments.
+ *
+ *   vopen() is an internal TizenRT interface and should not be called from
```

Comment:
If this function is internal, how about locating this above line 74 after recovering  /** Private Functions ***/?

Reply:
Currently, we are porting functions we need. To avoiding conflict, we follow the same sequence as Nuttx.